databricks-kube/src/context.rs

```diff
@@ -32,15 +32,24 @@ pub struct OperatorConfiguration {
 
 impl Context {
     pub fn get_databricks_url_token(&self) -> Option<(String, String)> {
-        let latest_secret = Self::latest_api_secret(self.api_secret_store.clone())?;
+        let latest_secret = Self::latest_store(self.api_secret_store.clone())?;
 
         let url = latest_secret.get("databricks_url")?;
         let token = latest_secret.get("access_token")?;
         Some((url.to_string(), token.to_string()))
     }
 
-    fn latest_api_secret(api_secret_store: Arc<Store<Secret>>) -> Option<BTreeMap<String, String>> {
-        api_secret_store
+    pub fn get_timeout_params(&self) -> Option<(String, String, String)> {
+        let latest_secret = Self::latest_store(self.api_secret_store.clone())?;
```

Comment:
Thinking about also renaming `api_secret_store` to just `var_store` and it'll be the general place where we put all our vars that we'd like to customize